models/vista2d/configs/metadata.json

```diff
@@ -0,0 +1,100 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json",
+    "version": "0.2.1",
+    "changelog": {
+        "0.2.1": "initial OSS version",
+        "0.2.0": "support bundle inference json",
+        "0.1.9": "finetuning checkpoint loading update",
+        "0.1.8": "added manfests",
+        "0.1.7": "prescaling to 0..1",
+        "0.1.6": "data processing script instead of json generation, data licenses",
+        "0.1.5": "improvements and mlflow support",
+        "0.1.4": "add unit tests and use monai 1.3.1",
+        "0.1.3": "update default optimizer",
+        "0.1.2": "use_weighted_sampler defaults to false",
+        "0.1.1": "remove unused code",
+        "0.1.0": "update evaluator",
+        "0.0.9": "do workflow_type and dataset data changes",
+        "0.0.8": "let pretrained_ckpt_name refer to ckpt_path",
+        "0.0.7": "update workflow input arg",
+        "0.0.6": "update the transforms and model, partly verified the reimplementation",
+        "0.0.5": "add inference logic and rename the entry point class from train.py to workflow.py",
+        "0.0.4": "update loadimage and weighted sampler",
+        "0.0.3": "fix final val error if override datalist",
+        "0.0.2": "fix attr error",
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "1.3.1",
```

Comment:
I guess this needs to be "1.4"

Reply:
Hi @mingxin-zheng . 1.3.1 is fine here. It's the initial version with customized components

Reply:
Hi @mingxin-zheng , when testing with CI dockerfile, I used back to monai 1.4, and update all other dependencies. Thanks for the suggestion